installations/InstallationsExample/ViewController.m

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "ViewController.h"
+
+// TODO: Remove once FirebaseInstallations released.
+#import <FirebaseInstallations/FIRInstallations.h>
+#import <FirebaseInstallations/FIRInstallationsAuthTokenResult.h>
+
+@import Firebase;
+
+@interface ViewController ()
+@property (strong, nonatomic) IBOutlet UIButton *getInstallationButton;
+@property (strong, nonatomic) IBOutlet UIButton *getAuthTokenButton;
+@property (strong, nonatomic) IBOutlet UIButton *deleteInstallationButton;
+@property (strong, nonatomic) IBOutlet UITextView *logTextView;
+
+@property(strong, nonatomic) NSString *log;
+@end
+
+@implementation ViewController
+
+- (void)dealloc {
+  [NSNotificationCenter defaultCenter];
```

Comment:
Is this missing a `removeObserver` call?

Reply:
Good catch!